bootstrap/cmd/bootstrap/app/options/options.go

```diff
@@ -47,6 +48,7 @@ func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
 	fs.BoolVar(&s.JsonLogFormat, "json-log-format", true, "Set true to use json style log format. Set false to use plaintext style log format")
 	fs.IntVar(&s.Port, "port", 8080, "The port to use when running an http server.")
 	fs.StringVar(&s.AppDir, "app-dir", "/opt/bootstrap", "The directory for the ksonnet applications.")
+	fs.StringVar(&s.GkeVersion, "GkeVersion", "1.10.6", "Initial GKE master version")
```

Comment:
Why is GkeVersion being set as a flag? Doesn't this mean default is now being set in multiple places; e.g. the DM config and the flag variable?

If we want to give users a way to override it can we surface it in the UI?

Reply:
With flag we can edit the default GKE version without changing image.
We can surface version in the UI, but seems not all GKE versions are equally stable, so for now how about set a stable version for both user and prober test so that the target GKE version is guaranteed to be tested?

Reply:
But if we change the default in the click to deploy UI but not the configs; won't the behavior in the app and via kfctl.sh diverge?